app/components/github-repo.js

```diff
@@ -11,12 +11,17 @@ export default Component.extend(RepoSyncProgressMixin, {
 
   githubRepo: null,
   project: null,
-  projectGithubRepo: null,
   showSettings: false,
 
+  belongsToProject: computed('projectGithubRepo.project.id', 'project.id', function() {
+    let repoProjectId = get(this, 'projectGithubRepo.project.id');
+    let projectId = get(this, 'project.id');
+    return repoProjectId === projectId;
+  }),
   isConnected: notEmpty('projectGithubRepo'),
   isLoading: alias('githubRepo.isLoading'),
   name: alias('githubRepo.name'),
+  projectGithubRepo: alias('githubRepo.projectGithubRepo.content'),
```

Comment:
No bueno.

Reply:
My guess is, because it's  some sort of proxy (probably a `PromiseProxy` `isConnected` above is not empty while `githubRepo.projectGithubRepo` is loading, then switches to empty when it's loaded, because then `content` is `null` and it proxies to `content`.

We could maybe fix this by having

```Javascript
isConnected: notEmpty('projectGithubRepo.id'),
projectGithubRepo: alias('githubRepo.projectGithubRepo')
```

Alternatively, some sort of combination of `projectGithubRepo.isFulfilled` and `notEmpty('projectGithubRepo')` for `isConnected`, but I'd rather we just checked `content` than that.